From 724d5edac76d8c9a4198b74c80286df38ed81679 Mon Sep 17 00:00:00 2001
From: Randy Dunlap <rdunlap@infradead.org>
Date: Fri, 15 Feb 2013 08:51:40 +0000
Subject: [PATCH] i2c: fix i2c-ismt.c printk format warning

Fix printk format warning.  dma_addr_t can be 32-bit or 64-bit,
so cast it to long long for printing.  This also matches the
printk format specifier that is already used.

drivers/i2c/busses/i2c-ismt.c:532:3: warning: format '%llX' expects argument of type 'long long unsigned int', but argument 4 has type 'dma_addr_t' [-Wformat]

Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Acked-by: Neil Horman <nhorman@tuxdriver.com>
Signed-off-by: Wolfram Sang <wolfram@the-dreams.de>

---
 drivers/i2c/busses/i2c-ismt.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/i2c/busses/i2c-ismt.c
+++ b/drivers/i2c/busses/i2c-ismt.c
@@ -530,7 +530,7 @@ static int ismt_access(struct i2c_adapte
 		}
 
 		dev_dbg(dev, " dma_addr = 0x%016llX\n",
-			dma_addr);
+			(unsigned long long)dma_addr);
 
 		desc->dptr_low = lower_32_bits(dma_addr);
 		desc->dptr_high = upper_32_bits(dma_addr);
